Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java

```diff
@@ -55,11 +58,16 @@ protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_main);
 
+
+    dataBaseOpenHelper = new DataBaseOpenHelper(this);
+
+
     Button btnFeedlyAccount = (Button) findViewById(R.id.FeedlyAccountBtn);
     btnFeedlyAccount.setOnClickListener(new Button.OnClickListener() {
       public void onClick(View v) {
         final String URL = "https://cloud.feedly.com/v3/streams/contents?streamId=user/" + ID + "/category/global.all";
-        new GetPageList().execute(URL);
+        new GetPageList(dataBaseOpenHelper).execute(URL);
```

Comment:
Can we just do `new DataBaseOpenHelper(this)` at here instead of `dataBaseOpenHelper`?
Is `dataBaseOpenHelper` member variable gonna be used on somewhere else?